src/head.ejs

```diff
@@ -24,17 +24,10 @@
         <% if(scope.description) { %>
         <meta name="twitter:description" content="<%= scope.description %>">
         <meta name="og:description" content="<%= scope.description %>">
-        <% } %>        
+        <% } %>
 
-        <!-- TODO: Concatenate -->
-        <script src="./jspm_packages/system.js"></script>
-        <script src="./js/config.js"></script>
-        <script>
-            /* global System */
-            // Must be loaded before the body is parsed?
-            System.import('js/modernizr');
-        </script>
-        <script src="./js/picturefill.min.js" async></script>
```

Comment:
Ahh, should've read this first—you have removed system+config. This means we lose the nice "refresh to reload" workflow in development. Could we keep this workflow whilst still bundling in production? (i.e. `jspm bundle --inject` https://github.com/jspm/jspm-cli/blob/master/docs/production-workflows.md#loading-a-bundle-automatically-inject)